pkgs/development/mobile/androidenv/ndk-bundle/default.nix

```diff
@@ -23,9 +23,6 @@ deployAndroidPackage {
     # to still support the old standalone toolchains builds.
     if [ -d $out/libexec/android-sdk/ndk ] && [ ! -d $out/libexec/android-sdk/ndk-bundle ]; then
         ln -sf $out/libexec/android-sdk/ndk/${package.revision} $out/libexec/android-sdk/ndk-bundle
-    else
-        echo "The ndk-bundle layout has changed. The nix expressions have to be updated!"
-        exit 1
```

Comment:
Something is wrong if this doesn't pass. The new ndk's have ndk folder in them, but no ndk-bundle

Reply:
Yes, that is true. But for NDK < 23, there is `ndk-bundle`, but no `ndk`. 
So if we take `22.1.7171670` for example, the condition for the `if` would not be fulfilled, as `ndk-bundle` exists. Then it would jump into the `else` and everything stops. In that case we should just ignore it?

Reply:
In that case I would do `elif [ ! -d $out/libexec/android-sdk-ndk-bundle ]` instead of removing this else branch all together. I did not test all the ndk versions but ndks older than 19 are no longer supported https://github.com/NixOS/nixpkgs/pull/160554#discussion_r810449633

Reply:
Done 👍